addons/web/static/src/core/datetime/datetime_picker.js

```diff
@@ -322,6 +322,7 @@ export class DateTimePicker extends Component {
         isDateValid: { type: Function, optional: true },
         dayCellClass: { type: Function, optional: true },
         tz: { type: String, optional: true },
+        noButtons: { type: Boolean, optional: true },
```

Comment:
It s not a good practice to use negation in the name of a variable. In this case, you can use "withButtons" and the default value is true